app/Controllers/feedController.php

```diff
@@ -136,7 +149,7 @@ public function addAction() {
 				// User want to create a new category, new_category parameter
 				// must exist
 				$new_cat = Minz_Request::param('new_category');
-				$new_cat_name = isset($new_cat['name']) ? $new_cat['name'] : '';
+				$new_cat_name = isset($new_cat['name']) && !empty(trim($new_cat['name'])) ? $new_cat['name'] : '';
```

Comment:
Good idea. But it would probably be good to apply trim() in all cases, to remove whitespaces at start and end.

Reply:
Hahaha, I was thinking about it, but wasn't unsure, alright give me 3 minutes